packages/react/src/tabs/tabs-list/TabsList.tsx

```diff
@@ -94,6 +92,7 @@ const TabsList = React.forwardRef(function TabsList(
         highlightedIndex={highlightedTabIndex}
         enableHomeAndEndKeys
         loop={loop}
+        orientation={orientation}
```

Comment:
Forgot this in https://github.com/mui/base-ui/pull/751
For `Tabs`, the cross-orientation arrow key should be ignored (and scroll the page if applicable)